addon/globalPlugins/EnhancedFindDialog/guiHelper.py

```diff
@@ -90,29 +90,32 @@ def __init__(self, parent, cursorManager, profile, searchEntries):
 
 		mainSizer = wx.BoxSizer(wx.VERTICAL)
 
-		findSizer = wx.BoxSizer(wx.HORIZONTAL)
+		sHelper = guiHelper.BoxSizerHelper(self, orientation=wx.VERTICAL)
 		# Translators: Dialog text for NvDA's find command.
 		textToFind = wx.StaticText(self, wx.ID_ANY, label=__("Type the text you wish to find"))
-		findSizer.Add(textToFind)
+		sHelper.addItem(textToFind)
 		self.findTextField = wx.ComboBox(self, wx.ID_ANY, choices = searchEntries,style=wx.CB_DROPDOWN)
-
+		sHelper.addItem(self.findTextField)
```

Comment:
2nd part of the suggestion:

```suggestion
		hSizer = wx.BoxSizer(wx.HORIZONTAL)
		hSizer.Add(textToFind, flag=wx.ALIGN_CENTER_VERTICAL)
		hSizer.AddSpacer(guiHelper.SPACE_BETWEEN_ASSOCIATED_CONTROL_HORIZONTAL)
		hSizer.Add(self.findTextField)
		sHelper.addItem(hSizer)
```